bitnami/superset/templates/ingress.yaml

```
@@ -42,7 +44,7 @@ spec:
             {{- if eq "true" (include "common.ingress.supportsPathType" $) }}
             pathType: {{ default "ImplementationSpecific" .pathType }}
             {{- end }}
-            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-web" (include "common.names.fullname" .)) "servicePort" "http" "context" $) | nindent 14 }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-web" (include "common.names.fullname" $root)) "servicePort" "http" "context" $) | nindent 14 }}
```

You can apply this suggestion and remove the line at the beginning.

```suggestion
            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-web" (include "common.names.fullname" $)) "servicePort" "http" "context" $) | nindent 14 }}
```